### DISCUSSION TITLE
Need assistance, a new contributor want's to dive into the community :)

### DISCUSSION THREAD

**Discussion by mohitdevelops** [reactplay/react-play#278] [Q&A]
Hey guys,
My name is Mohit, working with react as a beginner. I need some assistance regarding contributing to this wonderful community, I'm having a tiny issue while contributing. I have initialized the project on my local but how to work with this folder and how to push it to the repo?

![image](https://user-images.githubusercontent.com/51396626/172041288-4c1e54d9-8cba-4304-9f93-3b793f242517.png)

Thank you :)

**Comment by atapas** [comment]
@mohitdevelops 

You need to fork the repo

Create branch

Put your Code

Commit

Push

Create a Pull Request.



Feel free to ask if you get stuck.

**Reply by mohitdevelops** [reply]
Hi sir I've created a pull request, hope I did it right. 
Also I couldn't add preview image of my project, can you please tell me how to do that?
Thank you

**Reply by mohitdevelops** [reply]
Hi Sir, 
I've updated the files as per your review, now I'm trying to push the file and it's showing me this 👇
![image](https://user-images.githubusercontent.com/51396626/172176364-5378538a-7cbb-4980-976e-15750ed02906.png)

**Reply by atapas** [reply]
Hey @mohitdevelops 

We are setting up a Discord. Do you want to join and post your queries there?

https://discord.gg/RBu5EZBw

Feel free to post your questions under the need-help channel.
